### PR TITLE
priority_queue addition to this repo

### PR DESCRIPTION
**Pull request by srinchow**
hey ping me up incase of any modifications or changes .

### CODE REVIEW

File: priority_queue/priority_queue.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+int main ()
+{
+  int myints[]= {10,60,50,20};
+  std::vector<int> arr =  {1,2 3,4};
+  
+  //empty queue
+  std::priority_queue<int> first;
+  
+  //queue creation using array as input
+  std::priority_queue<int> second (myints,myints+4);
```
Comment (Delta456) [inline priority_queue/priority_queue.md:23]: It should be `std::priority_queue<int> second (myints, myints + 4);`

File: priority_queue/priority_queue.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+``` cpp
+
+#include <iostream>
+#include <queue>
+#include <vector>
+#include <functional>
+
+int main ()
+{
+  int myints[]= {10,60,50,20};
```
Comment (Delta456) [inline priority_queue/priority_queue.md:16]: Use `std::array<int> myints = {10, 60, 50, 20};`

File: priority_queue/priority_queue.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  std::vector<int> arr =  {1,2 3,4};
+  
+  //empty queue
+  std::priority_queue<int> first;
+  
+  //queue creation using array as input
+  std::priority_queue<int> second (myints,myints+4);
+  
+  //queue creation from vector
+  std::priority_queue<int> third ( arr.begin(),arr.end());
```
Comment (Delta456) [inline priority_queue/priority_queue.md:26]: It should be `std::priority_queue<int> third (arr.begin(), arr.end());`

File: priority_queue/pop.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    // inserts 20 as it is now the largest it comes to the first position
+    my.push(20);
+    
+    std::cout << my.top() << "\n";
+    
+    // removes the largest value i.e 20
+    my.pop();
+    
+    // largest is 10 now as 20 is removed
+    std::cout<<my.top()<<"\n";
```
Comment (Delta456) [inline priority_queue/pop.md:25]: Space here

File: priority_queue/pop.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    
+    // removes largest values i.e 10
+    my.pop();
+
+    // largest is 4 now
+    std::cout<<my.top();
```
Comment (Delta456) [inline priority_queue/pop.md:31]: Space here

File: priority_queue/priority_queue.md (hunk 2 above)
Comment (srinchow) [reply priority_queue/priority_queue.md]: `std:array<int,4> myints;` using this template

**Comment by srinchow** [comment]
changes were made as suggest please review it further.

File: priority_queue/priority_queue.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+#include <iostream>
+#include <queue>
+#include <vector>
+#include <functional>
+
+int main ()
+{
+  std::array<int, 4> myints = {10, 60, 50, 20};
+  std::vector<int> arr =  {1, 2, 3, 4};
```
Comment (Delta456) [inline priority_queue/priority_queue.md:17]: Please include array header file by `#include <array>`

File: .vscode/settings.json [hunk 7]
```diff
@@ -0,0 +1,3 @@
+{
+    "maven.view": "hierarchical"
+}
```
Comment (Delta456) [inline .vscode/settings.json:3]: Please remove this file

**Comment by srinchow** [comment]
Done with the changes

**Pull request merged**